### ISSUE TITLE
Deploy to NPM from Github Actions

### ISSUE THREAD

**Issue by owenpearson** [ably/ably-js#893]
Once #889 is merged we should update the GitHub workflow to also release to NPM.

**Comment by QuintinWillison** [comment]
@owenpearson I think this is a duplicate of #805. I'll leave it up to you and @Peter-Maguire to decide which to keep open, or both if you need the separation.
